app/src/pages/inside/dashboardPage/dashboardPage.jsx

```diff
@@ -89,6 +93,10 @@ const messages = defineMessages({
     addDashboard: addDashboardAction,
   },
 )
+@withPagination({
+  paginationSelector: dashboardPaginationSelector,
+  namespace: NAMESPACE,
+})
```

Comment:
Pagination controls does not have any effect on requests. Pls recheck this case.

Reply:
Also with your changes the search is disabled- it cannot be edited after typing the value.
The empty state is also wrong.
Your version.
![image](https://github.com/user-attachments/assets/5a2cc962-1884-49a3-9046-861c0fb6493e)
Develop version.
![image](https://github.com/user-attachments/assets/103d9194-42c9-4292-83e3-f5a8330e89a6)